PackageExplorer/SignPackageDialog.xaml.cs

```diff
@@ -0,0 +1,38 @@
+﻿using PackageExplorerViewModel;
+using System;
+using System.Windows;
+
+namespace PackageExplorer
+{
+    public partial class SignPackageDialog : StandardDialog
+    {
+        public SignPackageDialog()
+        {
+            InitializeComponent();
+        }
+
+        public string SignedPackagePath { get; private set; }
+
+        private void OnCertificatePasswordChange(object sender, EventArgs args)
+        {
+            var viewModel = (SignPackageViewModel)DataContext;
```

Comment:
Would it be better to just have a strongly-typed property on the dialog that does this? `ViewModel => (SignPackageViewModel)DataContext`?

Reply:
I can do that. I just copied it from https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/cdf3abacfc66363b49f6ccbb1968334aa8eb1948/PackageExplorer/PublishPackageWindow.xaml.cs#L23